libs/zzz/solver-pando/src/parentWorker.ts

```
@@ -0,0 +1,17 @@
+import type { ParentCommand } from '@genshin-optimizer/game-opt/solver'
+import { ParentWorker } from '@genshin-optimizer/game-opt/solver'
+
+const parentWorker = new ParentWorker(
+  () =>
+    new Worker(new URL('./childWorker.ts', import.meta.url), { type: 'module' })
+)
+
+// Receiving a message from main thread to worker
+onmessage = async (e: MessageEvent<ParentCommand>) => {
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)